lib/src/ovis_util/dstring.c

```diff
@@ -292,6 +292,8 @@ void dstr_trunc(dstring_t * dsPtr, int length)
 void dstr_free(dstring_t * dsPtr)
 {
 	assert(NULL != dsPtr);
```

Comment:
@baallan, I think this assert needs to be removed.

Reply:
ok